music21/stream/base.py

```diff
@@ -5448,6 +5458,7 @@ def toWrittenPitch(self, *, inPlace=False):
         * Changed in v5 returns `None` if `inPlace=True`
         '''
         from music21 import spanner
+        from music21 import stream
```

Comment:
not needed.  You can just do `assert isinstance(partLike, Stream)` within stream/base.py.

Reply:
done.